install.sh

```diff
@@ -77,21 +120,25 @@ for dir in "${CREW_CONFIG_PATH}/meta" "${CREW_DEST_DIR}" "${CREW_PACKAGES_PATH}"
   fi
 done
 
+echo -e "\n${YELLOW}Downloading information for Bootstrap packages...${RESET}"
+echo -en "${GRAY}"
```

Comment:
Is this line necessary?

Reply:
It is used to set the output color of `curl`